Documentation/Patterns.md

```diff
@@ -3,22 +3,22 @@
 ## Introduction
 
 Over time numerous reoccurring patterns have emerged from or were adopted by
-the serenity code base. This document aims to track and describe them, so they
+the Serenity code base. This document aims to track and describe them, so they
 can be propagated further and the code base can be kept consistent.
 
 ## `TRY(...)` Error Handling
 
-The `TRY(..)` macro is used for error propagation in the serenity code base.
+The `TRY(...)` macro is used for error propagation in the Serenity code base.
 The goal being to reduce the amount of boiler plate error code required to
 properly handle and propagate errors throughout the code base.
 
-Any code surrounded by `TRY(..)` will attempt to be executed, and any error
+Any code surrounded by `TRY(...)` will attempt to be executed, and any error
 will immediately be returned from the function. If no error occurs then the
-result of the contents of the TRY will be the result of the macro's execution.
+result of the contents of the `TRY(...)` will be the result of the macro's execution.
 
 ### Examples:
 
-Example from LibGfx:
+Example from [LibGfx](../Userland/Libraries/LibGfx/Bitmap.cpp):
```

Comment:
Let's not link to actual code. It adds close to nothing, and adds the risk of the link becoming stale.

Reply:
Done.